.editorconfig

```diff
@@ -18,6 +18,7 @@ indent_style = space
 indent_size = 2
 
 [*.hbs]
+insert_final_newline = false
 indent_style = space
 indent_size = 2
```

Comment:
https://github.com/softlayer/sl-ember-test-helpers/pull/72/files changed "indent_size" property throughout this file in that repo but the same changes were not made here.  We should be consistent across the repositories in this regard.